sms/v1/openapi.yaml

```
@@ -1008,7 +1008,7 @@ components:
           default: true
         timestamp:
           type: string
-          description: The timestamp when the message converted. If empty conversion timestamp will be set tp the timestamp of the event received in the API
+          description: The timestamp when the message has converted. If empty, the conversion timestamp will be set to the timestamp of the event received in the API
```

```suggestion
          description: The timestamp when the message was converted. If empty, the conversion timestamp is set to the timestamp of the event received in the API
```

Nebo ještě jde místo was hodit has been converted. Každopádně, přijde mi, že tady dává smysl mít trpný rod :)